.evergreen/start-kmip-server.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -x
+
+cd "$DRIVERS_TOOLS"/.evergreen/csfle || exit
+./activate-kmstlsvenv.sh
+
+# The -u options forces the stdout and stderr streams to be unbuffered.
+# TMPDIR is required to avoid "AF_UNIX path too long" errors.
+./activate-kmstlsvenv.sh
+TMPDIR="$(dirname "$DRIVERS_TOOLS")" python -u kms_kmip_server.py --ca_file ../x509gen/ca.pem --cert_file ../x509gen/server.pem  --port 5698 &
+python -u kms_http_server.py --ca_file ../x509gen/ca.pem --cert_file ../x509gen/expired.pem --port 8000 &
+python -u kms_http_server.py --ca_file ../x509gen/ca.pem --cert_file ../x509gen/wrong-host.pem --port 8001 &
+python -u kms_http_server.py --ca_file ../x509gen/ca.pem --cert_file ../x509gen/server.pem --port 8002 --require_client_cert &
+
+pgrep -f "kms_...._server.py" > kmip_pids.pid
```

Comment:
Does this pgrep command work on all the hosts we test on?

Reply:
I don't see any errors in my latest patches, so it seems so.